src/Compilers/Core/CodeAnalysisTest/ObjectSerializationTests.cs

```diff
@@ -35,17 +35,16 @@ private void TestInvalidStreamVersion()
 
         private void RoundTrip(Action<ObjectWriter> writeAction, Action<ObjectReader> readAction, bool recursive)
         {
-            var stream = new MemoryStream();
-            var writer = new ObjectWriter(stream);
-
-            writeAction(writer);
-            writer.Dispose();
+            using var stream = new MemoryStream();
 
-            stream.Position = 0;
-            using (var reader = ObjectReader.TryGetReader(stream))
+            using (var writer = new ObjectWriter(stream, leaveOpen: true))
             {
-                readAction(reader);
+                writeAction(writer);
             }
+
+            stream.Position = 0;
+            using var reader = ObjectReader.TryGetReader(stream);
```

Comment:
❔ Shouldn't this also leave the stream open, since `stream` is in its own `using` statement?

Reply:
It doesn't matter since both the stream and the reader are closed at the same time. I have not changed the default unless it was needed.